.dockerignore

```diff
@@ -4,7 +4,7 @@
 
 # Include specific files and folders:
 #
-!requirements/common.txt
+!requirements/*.txt
```

Comment:
Can you change to:

```
!requirements
```

Reply:
I was thinking that we might also include some documentation file in the folder (my opinion at this time is that we should document *why* packages are installed, so that if someone removes or reimplements a feature, it might be safer to remove the dependency).

If you feel strongly about this, I could change it.

Reply:
Good idea about an explanation of packages.

This change is pretty minor but keeps consistency with the other entries.

Reply:
If consistency is the goal, it is better to then explicitly include (in this case "not exclude") the common and dev .txt files.

!requirements/common.txt
!requirements/dev.txt

Would that make it better?

Reply:
Works for me.